src/components/SearchInput.vue

```diff
@@ -19,6 +19,7 @@
     modelValue: string | null | undefined,
     placeholder?: string,
     label?: string,
+    small?: boolean,
```

Comment:
Should this be a `size` prop to be consistent with the direction adam is going with prefect-design?

Reply:
Ah yeah, that'll require a small shim for small in the meantime but I'm ok with that.